internal/graph/check.go

```diff
@@ -70,7 +71,7 @@ func (r *ResolveCheckRequest) GetContextualTuples() []*openfgapb.TupleKey {
 
 func (r *ResolveCheckRequest) GetResolutionMetadata() *ResolutionMetadata {
 	if r != nil {
-		return r.ResolutionMetadata
```

Comment:
Please leave as is, this was purposefully designed to mimic an RPC stub so that it can be implemented as a internal gRPC interface.